pkg/app/run_test.go

```diff
@@ -790,6 +790,16 @@ COMMANDS
         --server-side-encryption-kms-key-id=SERVER-SIDE-ENCRYPTION-KMS-KEY-ID
                                  Server-side KMS Key ID. Must be set if
                                  server-side-encryption is set to aws:kms
+        --compression-codec=COMPRESSION-CODEC
+                                 The codec used for compression of your logs.
+                                 Valid values are zstandard, snappy, and gzip.
```

Comment:
Let's update `zstandard` to be `zstd` (across this entire file) as per the latest update to the API reference.

Reply:
Addressed by https://github.com/fastly/cli/pull/190/commits/fda2657dc6aa05f1570de57996c68b73f62aadc3